src/components/Square.css

```diff
@@ -1,12 +1,33 @@
 .square
 {
-  border: 4px solid #2c3e50;
+  border: 3px solid black;
```

Comment:
LOVE the css hover effects! The overall color palette is super cute and portfolio ready ✨